cc.cozycloud.errors/request

```diff
@@ -0,0 +1,4 @@
+POST https://errors.cozycloud.cc/api/{{project}}/store/
+X-Sentry-Auth: Sentry sentry_version={{sentry_version}}, sentry_client={{sentry_client}}, sentry_timestamp=1503522542, sentry_key={{sentry_key}}
```

Comment:
Are you sure sentry_timestamp should be a fixed value?

Reply:
see https://develop.sentry.dev/sdk/overview/#authentication

> sentry_timestamp
    The unix timestamp representing the time at which this event was generated.

Reply:
Je ne sais pas, c'est un copier coller de l'ancien code. @Crash-- ?

Reply:
Les 2 PR initiales sur le sujet https://github.com/cozy/cozy-doctypes/pull/16 et https://github.com/cozy/cozy-doctypes/pull/92 peut-être que @nono a plus d'historique ?

Reply:
Non, je n'en sais pas plus

Reply:
Je n'ai pas réussi via les url officiels (site + github) à retomber sur la doc que tu pointes, est-ce que ce ne serait pas une ancienne doc ?

A mon avis c'est un param qui ne sert plus : 
- Il n'est pas présent dans la partie `auth` de la doc : https://docs.sentry.io/clients/javascript/config/
- Il a été supprimé du SDK JS officiel : https://github.com/getsentry/sentry-javascript/blob/master/CHANGELOG.md#5130 , https://github.com/getsentry/sentry-javascript/pull/2458
- Il n'y a plus aucune de référence à `sentry_timestamp` dans le code du SDK JS officiel : https://github.com/getsentry/sentry-javascript/search?q=sentry_timestamp

Je corrige la PR en fonction.

Reply:
bien vu merci ! J'ai pas poussé assez la partie API de la doc...

Reply:
vu la PR sur le SDK js officiel, j'en conclus que le sentry_timestamp n'est effectivement plus utilisé.